utils/schemed-kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/schemedkafka/schemaregistry/confluent/schemaid/SchemaIdFromNuHeadersAndPotentiallyConfluentPayload.scala

```diff
@@ -0,0 +1,36 @@
+package pl.touk.nussknacker.engine.schemedkafka.schemaregistry.confluent.schemaid
+
+import org.apache.kafka.common.header.Headers
+import pl.touk.nussknacker.engine.schemedkafka.schemaregistry.{GetSchemaIdArgs, SchemaId, SchemaIdFromMessageExtractor, SchemaIdWithPositionedBuffer}
+import pl.touk.nussknacker.engine.schemedkafka.schemaregistry.confluent.ConfluentUtils
+
+import java.nio.ByteBuffer
+import scala.util.Try
+
+object SchemaIdFromNuHeadersAndPotentiallyConfluentPayload extends SchemaIdFromMessageExtractor with Serializable {
```

Comment:
Maybe let's name it to `SchemaIdFromNuHeaders` and leave a comment, that even though we take schemaId from headers it can be also in payload - so bytearray must be shifted?


If you do not want to rename, I think a comment would be still welcomed, cause it's not so obvious.

Reply:
I'd rather like to keep confluent in name to make this Confluent logic more explicit but:
- I changed the name to be visible so that it only shifts payload - not extract schema id from it
- I added verbose comment why it is done this way
WDYT?

Reply:
this name is a lot better, and a comment explains everything, thanks